app/lnurlp/[username]/route.ts

```diff
@@ -117,9 +141,9 @@ export async function GET(
   let minSendable = 1000 // 1 sat in millisat
   let maxSendable = 100000000000 // 1 BTC in millisat
 
-  if (amount && Number.isInteger(Number(amount))) {
-    minSendable = Number(amount) * 1000
-    maxSendable = Number(amount) * 1000
+  if (amountInMsats) {
+    minSendable = amountInMsats
```

Comment:
shouldn't we update invoice expiration when we have a currency different to the wallet currency?

Reply:
we should, but I don't think the BtcWallet have this ability right now?